subsys/bootloader/include/dummy_values/pm_config.h

```diff
@@ -1,14 +1,14 @@
 /* DUMMY FILE ONLY TO BE USED FOR TESTING */
 #ifndef PM_CONFIG_H__
 #define PM_CONFIG_H__
-#define PM_B0_ADDRESS 0x0
-#define PM_B0_SIZE 0x8000
-#define PM_B0_IMAGE_ADDRESS 0x8000
-#define PM_B0_IMAGE_SIZE 0x1000
-#define PM_B0N_ADDRESS 0x0
-#define PM_B0N_SIZE 0x8000
-#define PM_B0N_IMAGE_ADDRESS 0x8000
-#define PM_B0N_IMAGE_SIZE 0x1000
+#define PM_B0_PROVISION_ADDRESS 0x0
```

Comment:
squash with two previous commits.
I assume you cannot bisect over those three commit, as I assume the changes in this commit is required for successfully compiling the previous two commits.